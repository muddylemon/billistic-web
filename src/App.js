import React from "react"
import CssBaseline from '@material-ui/core/CssBaseline'
import "./app.scss"

export const App = () => {
  return (
    <React.Fragment>
      <CssBaseline />
      <div>
        And now we're cooking with gas
      </div>
    </React.Fragment>
  )
}
