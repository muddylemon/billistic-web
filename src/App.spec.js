import React from "react"
import { App } from "./App"
import ShallowRenderer from "react-test-renderer/shallow"

describe('App', () => {
  it('should render without crashing', () => {
    const tree = new ShallowRenderer().render(<App />)
    expect(tree).toMatchSnapshot()
  })
})
