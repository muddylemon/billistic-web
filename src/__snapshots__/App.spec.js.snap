// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render without crashing 1`] = `
<React.Fragment>
  <WithStyles(CssBaseline) />
  <div>
    And now we're cooking with gas
  </div>
</React.Fragment>
`;
